managed-databases/postgresql-and-mysql/api-cli/import-data-to-managed-postgresql-databases.mdx

```diff
@@ -0,0 +1,464 @@
+---
+meta:
+  title: Importing data into Managed Databases for PostgreSQL
+  description: This page explains how to import your data into Managed Databases for PostgreSQL from another database or backup files
+content:
+  h1: Importing data into Managed Databases for PostgreSQL
+  paragraph: This page explains how to import your data into Managed Databases for PostgreSQL from another database or backup files
+dates:
+  validation: 2024-06-24
+  posted: 2024-06-24
+---
+
+Managed Databases for PostgreSQL offers near-complete PostgreSQL compatibility, and allows you to import your data from any PostgreSQL-compatible database.
+
+<Message type="important">
+The methods listed below are best suited for database sizes below 100 GB (approx.). Above this size, we recommend you use dedicated tooling to import your data, as your local storage size and bandwidth will create bottlenecks, and the operation may last several hours.
+</Message>
+
+<Macro id="requirements" />
+
+- A Scaleway account logged into the [console](https://console.scaleway.com)
+- [Owner](/identity-and-access-management/iam/concepts/#owner) status or [IAM permissions](/identity-and-access-management/iam/concepts/#permission) allowing you to perform actions in the intended Organization
+
+- [Created a target Serverless SQL Database](/serverless/sql-databases/how-to/create-a-database) to import your data into.
+
+- Checked that the target database you are importing data into is empty, or contains data you can overwrite.
+
+The import procedure depends on your data source:
+    - [Importing data from an existing PostgreSQL database](#importing-data-from-an-existing-postgresql-database)
+      - [Using pg_dump](#using-pgdump)
+      - [Using pgAdmin](#using-pgadmin)
+      - [Using Airbyte](#using-airbyte)
+      - [Using Fivetran](#using-fivetran)
+      - [Using Meltano](#using-meltano)
+    - [Importing data from an existing PostgreSQL backup](#importing-data-from-an-existing-postgresql-backup)
+    - [Importing data from a file (.csv, .txt, etc.)](#importing-data-from-a-file-csv-txt-etc)
+
+## Importing data from an existing PostgreSQL database
+
+### Using pg_dump
+
+#### Prerequisites
+
+To complete this procedure, you must have installed PostgreSQL 16 (or newer) with [pg_dump](https://www.postgresql.org/docs/current/app-pgdump.html) and [pg_restore](https://www.postgresql.org/docs/current/app-pgrestore.html) (bundled with the default PostgreSQL installation).
```

Comment:
This is true for Serverless Databases as PG 16 is the only version supported but on Managed PostgreSQL, we don't support PG 16 yet... 

We have to adapt a little bit to say that the user must have installed PostgreSQL with pg_dump and pg_restore in the same version as the one of their Managed PostgreSQL

Reply:
Specifically, they must install pg_dump and pg_restore compatible with their target Managed PostgreSQL version.